doc/examples/compositing/contactSheet.gfr

```diff
@@ -0,0 +1,717 @@
+import Gaffer
+import GafferAppleseed
+import GafferDispatch
+import GafferImage
+import GafferOSL
+import GafferScene
+import IECore
+import imath
+
+Gaffer.Metadata.registerValue( parent, "serialiser:milestoneVersion", 0, persistent=False )
+Gaffer.Metadata.registerValue( parent, "serialiser:majorVersion", 54, persistent=False )
+Gaffer.Metadata.registerValue( parent, "serialiser:minorVersion", 0, persistent=False )
+Gaffer.Metadata.registerValue( parent, "serialiser:patchVersion", 0, persistent=False )
+
+__children = {}
+
+parent["variables"].addChild( Gaffer.CompoundDataPlug.MemberPlug( "projectName", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
```

Comment:
Are you copying the python from the .gfr file opened as text (or from GitHub)? If thats the case, I think that would be true of any Gaffer file rather than specifically the examples - these are just saved using the normal new -> save semantics. Is this much of an issue in common use do you think? When you copy-paste nodes in the UI it leaves out that stuff.

Reply:
It is true of any Gaffer file - the official way to "paste" an entire script into another is to use File->Import, which will ignore the clashing variables. This isn't something we need to concern ourselves with for this PR.

Reply:
I never noticed! I would guess because I've been copying directly from the graph this whole time.